spec/queries/users/get_user_spec.rb

```diff
@@ -17,6 +17,12 @@ def query(id:)
             email
             availabilities {
               id
+              userId
+              startDateTime
+              endDateTime
+              status
+              createdAt
+              updatedAt
```

Comment:
Here's the changes from this PR: all of the availability fields are being requested and tested.